projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -187,9 +188,14 @@ export class PdfViewerComponent implements AfterContentInit, OnChanges, OnDestro
 
   onMouseUp(mouseEvent: MouseEvent) {
     if (this.toolbarEvents.highlightModeSubject.getValue()) {
-      setTimeout(() => this.viewerEvents.textSelected({
-            page: this.pdfWrapper.getPageNumber(), event: mouseEvent
-      }), 0);
+      this.highlighting = true;
+      setTimeout(() => {
+        this.viewerEvents.textSelected({
+          page: this.pdfWrapper.getPageNumber(),
+          event: mouseEvent
```

Comment:
what about changing the padding value on the target of the event, or would that basically do the same thing?!

Reply:
the target event doesnt effect the whole selection, just the last mouse event.